pkg/multicluster/cluster_metrics_management.go

```diff
@@ -37,16 +40,24 @@ type ClusterMetricsHelper interface {
 }
 
 // NewClusterMetricsMgr will create a cluster metrics manager
-func NewClusterMetricsMgr(kubeClient client.Client) (*ClusterMetricsMgr, error) {
+func NewClusterMetricsMgr(kubeClient client.Client, refreshPeriod time.Duration) (*ClusterMetricsMgr, error) {
 	mgr := &ClusterMetricsMgr{
 		kubeClient: kubeClient,
 	}
-	err := mgr.Refresh()
-	return mgr, err
+	go func() {
```

Comment:
It would be better to pass a context value inside and use a `select` to check if the context is ended. If ends, this goroutinue can return back.

Reply:
OK, I will add it.

Reply:
@JooKS-me It is recommended to define an independent function.

Reply:
@barnettZQG @Somefive ok, I will modify it later.